.github/workflows/rust-ci.yml

```diff
@@ -9,6 +9,7 @@ on:
       - main
       - cloud-bridge
     paths:
+      - 'car-bridge/**'
```

Comment:
Isn't this unrelated to this PR?

Reply:
Unrelated, yes, but it became apparent with this PR as now we have two unit tests as part of the Car Bridge. I can PR it separately, or we can keep it as it is. No strong preference.

Reply:
Let's stick with consistently submitting unrelated changes as separate PRs that are easier to track.

Reply:
#66